civet.dev/cheatsheet.md

```diff
@@ -315,3 +315,39 @@ floor # / 2
 ```
 
 </div>
+
+## ASCII Symbols
+
+Here is a list of ASCII symbols and their various contextual meanings in Civet.
+(There are also more [Unicode symbols](/reference#unicode-operators).)
+
+| Symbol | Meanings |
+|--------|----------|
+| space | Function call |
+| `?` | Non-null test `x?`; non-null access `x?.y`, `x?y`, `x?(...)`, `x?[...]`; null coalescing `x ?? y` (`x ? y` with `coffeeBinaryExistential`); TypeScript optionals `(x?: number) => x`; if/then/else ternary `x ? y : z` |
+| `!` | Negation `!x`; object flag `{!x}`; TypeScript non-null assertion `x!`; non-null assertion access `x!.y`, `x!y` |
```

Comment:
Non-nullable types `let x: T!`, force-cast `as!`

Should we also mention that it can be interleaved with certain operators like `!in`?